### PR TITLE
add ur for Node.js link for all the other files

### PR DESCRIPTION
**Pull request by ghost**
Ref: https://github.com/nodejs/nodejs.org/pull/3687.

### CODE REVIEW

**Comment by MaledongGit** [comment]
@nodejs/nodejs-cn, @nodejs/nodejs-tr: Please have a review, thanks!

**Comment by ghost** [comment]
@nodejs/nodejs-cn, @nodejs/nodejs-tw : Please have a review, thanks!

**Comment by ghost** [comment]
PS: After this is merged, I'll notify the other groups to tranlate with the latest version of documents.

**Review by willin — APPROVED** [review]

**Review by nschonni — APPROVED** [review]

File: locale/ar/get-involved/index.md [hunk 1]
```diff
@@ -31,3 +31,4 @@ layout: contribute.hbs
 * [مجموعة المستخدمين اليابانية](https://nodejs.jp/)
 * [مجموعة الفيسبوك باللغة الأسبانية للـ Node.js](https://www.facebook.com/groups/node.es/)
 * [Vietnamese Node.js community](https://www.facebook.com/nodejs.vn/)
+* [Uzbekistan group for Node.js](https://t.me/nodejs_uz)
```
Comment (Aissaoui-Ahmed) [inline locale/ar/get-involved/index.md]:
```suggestion
* [مجتمع Node.js الفيتنامي](https://www.facebook.com/nodejs.vn/)
* [المجموعة الأوزبكستانية لـ Node.js](https://t.me/nodejs_uz)
```

**Comment by ghost** [comment]
@nodejs/website：Please update your translation, thanks.

**Pull request merged**